ddl/db_test.go

```diff
@@ -410,22 +410,46 @@ func (s *testSerialDBSuite) TestAddExpressionIndexRollback(c *C) {
 	ctx.Store = s.store
 	times := 0
 	hook.OnJobUpdatedExported = func(job *model.Job) {
-		if job.SchemaState == model.StateDeleteOnly {
+		switch job.SchemaState {
+		case model.StateDeleteOnly:
+			if job.SchemaState == model.StateDeleteOnly {
+				if checkErr != nil {
+					return
+				}
+				_, checkErr = tk1.Exec("delete from t1 where c1 = 40;")
```

Comment:
Could we add insert and update statements here? This schema state will be running two times.